lib/config.go

```diff
@@ -289,7 +289,10 @@ type NetworkConfig struct {
 	NetworkDir string `toml:"network_dir"`
 
 	// PluginDir is where CNI plugin binaries are stored.
-	PluginDir []string `toml:"plugin_dir"`
+	PluginDir string `toml:"plugin_dir,omitempty"`
```

Comment:
I think we break the backwards compatibility here since we convert the array into a string, right?

Reply:
We already broke backwards compatibility by changing the string to an array. This is reverting the change (which only exists in 1.14.0-1.14.2), so that we are more backwards compatible with versions older than 1.14.0. So yes, but backwards compatible with a small range.

Reply:
Then I'm totally fine with that. :)